wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -37,25 +37,26 @@ public static function getSubscribedEvents() {
    * @param object $objectRef
    *   The parameters that were sent into the calling function.
    */
-  public static function setOfficeDetails(string $op, string $objectName, int $objectId, &$objectRef) {
+  public static function setOfficeDetails(string $op, string $objectName, int $contactId, &$objectRef) {
 
-    if ($objectName !== 'AfformSubmission') {
+    if ($op !== 'create' || $objectName !== 'Address') {
```

Comment:
can we add one more check here to check the form or something unique because I expect the other things won't work if we merge it like this?
**Use case**: Please check the monitory contribution case here, and check if the new individual came there then it goes to their correct group.

**NOTE:** Please don't merge this PR before resolving these things and please test all the cases on staging after taking latest staging pull